src/components/ebay-menu-button/examples/20-prefix-id/template.marko

```diff
@@ -0,0 +1,8 @@
+<div id="custom-label">
+    Menu Label
+</div>
+<ebay-menu-button prefix-id="custom-label" text="eBay Menu">
```

Comment:
Same as previous example, can you make sure the label text is updated to reflect the current selection. If the button label text never updates, then an external label shouldn't be used, so I want to make sure the examples are good.